react/components/AppStoreCondition.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { Fragment, useEffect } from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+import { useIsUserLoggedIn } from '../hooks/useIsUserLoggedIn'
+
+interface Props {
+  redirectPath: string
+  authenticatedOnly: boolean
+}
+
+const AppStoreCondition: React.FC<Props> = ({
```

Comment:
Acho que não dá pra deduzir o que esse componente faz só lendo o nome. Eu colocaria algum nome que tem `login` no meio:

```suggestion
const RedirectIfNotLoggedIn: React.FC<Props> = ({
```

ou 

```suggestion
const WithUserLoggedIn: React.FC<Props> = ({
```

(Lembrar de atualizar os nomes dos arquivos se renomear)

Reply:
Mas o problema é que esse bloco n bloqueia apenas usuários logados. Ele pode bloquear qualquer usuário.

Reply:
Tipo, /cart

Reply:
Meu ponto é que o nome AppStoreCondition é genérico, não dá pra entender o que esse componente faz lendo o nome